src/graph/graphcommandexecutor.pas

```diff
@@ -127,6 +145,8 @@ procedure TGraphCommandExecutor.UpdateChartTitles(ST: TCustomGraphCommand;
     if (inc > 0) then
       Chart.Title.Font.Size := Chart.Title.Font.Size + inc;
 
+    ST.HasOption(['s', 'save', 'e', 'export'], Opt);
```

Comment:
This statement does nothing since the **Opt** variable is overwritten in the line below.

Reply:
removed